packages/url/src/prepend-http.js

```diff
@@ -24,7 +24,7 @@ export function prependHTTP( url ) {
 
 	url = url.trim();
 	if ( ! USABLE_HREF_REGEXP.test( url ) && ! isEmail( url ) ) {
-		return 'http://' + url;
+		return 'https://' + url;
```

Comment:
This is a potentially breaking change that needs to be documented as one in the package's CHANGELOG.